numpy/core/src/multiarray/dragon4.h

```diff
@@ -40,6 +40,45 @@
 #include "npy_pycompat.h"
 #include "numpy/arrayscalars.h"
 
+/* Half binary format */
```

Comment:
No, I think only dragon4 cares about the floating-point binary formats.

Reply:
In that case it might be better to put them in the C file. That could be part of future cleanups, as I suspect this is something of a continuing project :)